docker-compose.yaml

```diff
@@ -40,6 +40,7 @@ services:
       # @TODO sign instructions in https://grafana.com/docs/grafana/latest/developers/plugins/sign-a-plugin/
       # - GF_PLUGINS_ALLOW_LOADING_UNSIGNED_PLUGINS=vertamedia-clickhouse-datasource
       - GF_INSTALL_PLUGINS=grafana-piechart-panel,grafana-worldmap-panel
+      - GF_APP_MODE=development
```

Comment:
Not sure if this should be enabled by default

Reply:
which problem do you try to resolve to add `development` here?

Reply:
This was to load the plugin but it seems that the signing is working now so I do not need this flag. I have removed.